tf2onnx/graph.py

```diff
@@ -803,23 +803,17 @@ def update_node_shape_dtype(self, node, override=False):
                 logger.debug("Inferred dtype for [%s, type: %s] is UNDEFINED, SKIP", node.name, node.type)
             else:
                 existing_dtype = self.get_dtype(output)
-                if existing_dtype is not None and existing_dtype != dtype:
-                    if override:
-                        logger.warning("Override dtype of %s from %s to %s", output, existing_dtype, dtype)
-                    else:
-                        dtype = existing_dtype
+                if existing_dtype is not None and existing_dtype != dtype and not override:
+                    dtype = existing_dtype
```

Comment:
We only set `override=True` when we expect a difference, so no warning is needed.